js/containers/portal-dashboard/portal-dashboard-app.tsx

```diff
@@ -69,11 +70,12 @@ class PortalDashboardApp extends React.PureComponent<IProps, IState> {
     this.state = {
       initialLoading: true,
       scrollLeft: 0,
-      showAllResponsesPopup: false
+      showAllResponsesPopup: false,
+      authError: ""
     };
   }
 
-  componentDidMount() {
+  async componentDidMount() {
```

Comment:
`async` doesn't seem to be necessary here, there's no async code in the body of this function (no `await`).